### DISCUSSION TITLE
How to use Wiki.js class="links-list"?

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6598] [Help / Questions] [answered]
Hi everyone,

I'll start this post of by saying I have zero experience with web design, CSS, or HTML. Basically, I'd like to copy the format for inserting links in a page that is that is used on the wiki.js documentation site. How can this be accomplished? Below is an example of the format I'd like to mimic:

![image](https://github.com/requarks/wiki/assets/134225172/cf888ca0-158b-4b42-a2a7-fa10185999f8)

I tried to determine what the element is use is by "inspecting" the page. It looks like the element I need is called class="links-list"

**Comment by P4mes4n** [comment, marked as answer]
https://github.com/requarks/wiki/discussions/6484#discussioncomment-6107609
